README.md

```diff
@@ -0,0 +1,49 @@
+# Pixelpact 📸
+
+Pixeplact gets you started quickly with visual testing for the web.
+
+In contrast to functional or snapshot testing, visual testing verifies that the actually rendered HTML/CSS/JS of your web app truly looks the way it is supposed to. Pixelpact does this by taking screenshots and comparing them to reference images stored in your git repository (using [Git LFS](https://git-lfs.github.com/)).
+
+Checkout our [Getting Started Guide](./getting-started.md).
+
+## Goals
+
+Visual testing is not a new idea and there are plenty of other tools, libraries and service providers that offer a similar functionality as we do.In fact, Pixeplact took a lot of inspiration from existing solutions and is based on their hard work! We created Pixelpact with the following goals:
+
+* **Low entry barrier**. We created Pixeplact to provide a solution to get you started quickly with visual testing, without any vendor lock-in or service provider discussions. Start using Pixelpact today and start growing with visual testing.
```

Comment:
- typo: `Pixeplact`
- rewording suggestion: Pixelpact helps you to easily integrate visual testing, without any [...]